test/e2e/storage/drivers/csi.go

```diff
@@ -186,6 +186,9 @@ func (g *gcePDCSIDriver) GetDynamicProvisionStorageClass(fsType string) *storage
 	suffix := fmt.Sprintf("%s-sc", g.driverInfo.Name)
 
 	parameters := map[string]string{"type": "pd-standard"}
+	if fsType != "" {
+		parameters["fsType"] = fsType
```

Comment:
I don't think we specify fsType in the storage class for GCE PD, it's actually passed in on `NodeStageVolume` through the `VolumeCapability` field that comes from the `PVSource`.

Reply:
IIUC, CSI provisioner will [add fsType to PVC]( https://github.com/kubernetes-csi/external-provisioner/blob/master/pkg/controller/controller.go#L368) if fsType is set in StorageClass on provision. So, fsType should also be passed to PV, as a result, it will be passed as a ```VolumeCapability``` on ```NodeStageVolume``` when attach is requested.

Reply:
oh cool, I didn't know that, thanks for the explaination!

Reply:
@davidz627 this will still cause pd csi driver to fail right?  Should we be using the new namespaced fstype parameter?

Reply:
/hold 
My understanding is PD csi tests in serial will start failing as is